crates/solidity/outputs/typescript/lib/.cargo/config.toml

```diff
@@ -1,7 +1,3 @@
-[target.aarch64-unknown-linux-gnu]
-linker = "aarch64-linux-gnu-gcc"
 [target.aarch64-unknown-linux-musl]
```

Comment:
Are these settings generated by `napi init`? I wonder how are they can be kept up to date. I will need to merge all of these along with our monorepo config.

Reply:
yes they are.

Reply:
No, but I'm just updating from the latest github, not doing the "why?" in this PR.

Reply:
Probably because those specific linker exceptions are no longer required.

Reply:
My goal is to keep this as close as possible to napi head because we will want to stay up to date going foward, and having easy diffs from a fresh project init makes things easier.

Reply:
I'm fine with the changes in this PR, but from what I can see, not sure if we can "stay up to date going foward", as this command assumes it is a single top-level github repo/project. Some of these files are harmful, some conflict with our monorepo, and some need to be absorbed/merged with existing files.

I will look up what other NAPI users do in this case. Thanks!